scripts/system/onFirstRun.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+//
+//  onFirstRun.js
+//
+//  Created by Kalila L. on Oct 5 2020.
+//  Copyright 2020 Vircadia contributors.
+//
+//  This script triggers on first run to perform bootstrapping actions.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+(function() { // BEGIN LOCAL_SCOPE
+    var SETTING_TO_CHECK = 'firstRun';
+    var DEFAULT_NAME = '';
```

Comment:
```suggestion
    var DEFAULT_DISPLAY_NAME = '';
```
More descriptive name just in case some other name item is added to the script in the future.